src/components/activity/screens/ActivityAdd/views/ActivityAddView.js

```diff
@@ -67,10 +67,10 @@ export default class ActivityAddView extends React.Component {
             <ActivityAddSpecificFieldsContainer
               type={this.props.type}
               setContentSource={this.props.setContentSource}
-              setMeetingWho={this.props.setMeetingWho}
+              setEventWhat={this.props.setEventWhat}
```

Comment:
This property does not exists anymore in the parent component, and is no passed in the child component. 
I think this field has been removed in a previous commit, it does not exists in our databse.